dist/images/kubectl-ko

```diff
@@ -566,13 +566,13 @@ diagnose(){
 }
 
 getOvnCentralPod(){
-    NB_POD=$(kubectl get pod -n $KUBE_OVN_NS -l ovn-nb-leader=true | grep ovn-central | head -n 1 | awk '{print $1}')
+    NB_POD=$(kubectl get pod -n $KUBE_OVN_NS -l ovn-nb-leader=true | grep ovn-central | head -n 1 | awk '{if($2=="1/1" && $3=="Running") print $1}')
```

Comment:
`head -n -1` will return only the first line

Reply:
fixed